src/api/index.ts

```diff
@@ -11,5 +11,6 @@ export * from './locator'
 export * from './modules'
 export * from './table'
 export * from './verbose'
-export { BatchStream } from './typings'
+export * from './lib'
```

Comment:
How about we move the ./lib/constants to ./constants? Then here we would export `./constants'